### PR TITLE
Adapt finaid content from 2023 for 2024

### PR DESCRIPTION
**Pull request by cmaureir**
Initial change

### CODE REVIEW

**Comment by SebastiaanZ** [comment]
Hi, thank you, I'll will change a few things now.

**Comment by SebastiaanZ** [comment]
I have an update for this branch, but I don't have permissions to add commits to this branch (or the repo).

**Comment by europython-society** [comment]
> I have an update for this branch, but I don't have permissions to add commits to this branch (or the repo).

Hey @SebastiaanZ, you should have write permissions, could you verify that you cannot commit to the branch?

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ dependencies:
     version: 2.1.0
   date-fns:
     specifier: ^2.30.0
-    version: 2.30.0
+    version: 2.30.0f
```
Comment (cmaureir) [inline pnpm-lock.yaml:19]:
```suggestion
    version: 2.30.0
```
maybe this works?

**Pull request merged**